instrumentation/openai/openai-java-1.1/library/src/main/java/io/opentelemetry/instrumentation/openai/v1_1/OpenAITelemetryBuilder.java

```diff
@@ -53,7 +56,18 @@ public OpenAITelemetry build() {
             .addOperationMetrics(GenAiClientMetrics.get())
             .buildInstrumenter();
 
+    Instrumenter<EmbeddingCreateParams, CreateEmbeddingResponse> embeddingsInstrumenter =
+        Instrumenter.<EmbeddingCreateParams, CreateEmbeddingResponse>builder(
+                openTelemetry,
+                INSTRUMENTATION_NAME,
+                GenAiSpanNameExtractor.create(EmbeddingAttributesGetter.INSTANCE))
+            .addAttributesExtractor(
+                GenAiAttributesExtractor.create(EmbeddingAttributesGetter.INSTANCE))
+            .addOperationMetrics(GenAiClientMetrics.get())
+            .buildInstrumenter(SpanKindExtractor.alwaysClient());
```

Comment:
I realized I had the wrong span kind. I will fix / assert for chat spans in a separate PR